core/src/main/java/nl/nn/adapterframework/pipes/IteratingPipe.java

```diff
@@ -349,14 +349,14 @@ public StopReason handleItem(I item) throws SenderException, TimeoutException, I
 					}
 				} catch (SenderException e) {
 					if (isIgnoreExceptions()) {
-						log.info("ignoring SenderException after excution of sender for item ["+item+"]",e);
+						log.info("ignoring SenderException after execution of sender for item ["+item+"]",e);
```

Comment:
Misschien dat je gelijk:
```suggestion
						log.info("ignoring SenderException after execution of sender for item [{}]", item, e);
```